sdk/core/Azure.Core/src/Shared/AutoRest/PageableHelpers.cs

```diff
@@ -4,13 +4,26 @@
 #nullable enable
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core.Pipeline;
 
 namespace Azure.Core
```

Comment:
This type is to support code generation and the original copy is in https://github.com/Azure/autorest.csharp/blob/feature/v3/src/assets/Generator.Shared/PageableHelpers.cs 

Automatic CI pipeline overwrites this file when generator is updated so changes to it should be done in autorest.csharp repo.

Reply:
Ok, thanks.

Reply:
Let's discuss it here and move to autorest after. It's easier to see the impact in this repo.